python/lsst/pex/config/config.py

```diff
@@ -257,9 +257,9 @@ def save(self, outfile, instance):
         doc = "# " + str(self.doc).replace("\n", "\n# ")
         if isinstance(value, float) and (math.isinf(value) or math.isnan(value)):
             # non-finite numbers need special care
-            outfile.write("{}\n{}=float('{!r}')\n\n".format(doc, fullname, value))
+            outfile.write(u"{}\n{}=float('{!r}')\n\n".format(doc, fullname, value))
```

Comment:
Yes. This was driven by things working on py3 and then finding they did not work on py2.